packages/generate-pdf/test/generate-pdf.spec.ts

```diff
@@ -152,18 +157,6 @@ const generatePaths = (
   };
 };
 
-const waitWritableStreamEnd = (writeStream: Writable, outputFilePath: string) => {
-  return new Promise((resolve, reject) => {
-    writeStream
-      .on('finish', () => {
-        expect(existsSync(outputFilePath)).toEqual(true);
-        unlinkSync(outputFilePath);
```

Comment:
Removing this means that the output PDF files are no longer deleted. They are output to the root directory.